google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/StreamWriter.java

```diff
@@ -29,15 +29,11 @@
 import com.google.api.gax.core.InstantiatingExecutorProvider;
 import com.google.api.gax.grpc.GrpcStatusCode;
 import com.google.api.gax.retrying.RetrySettings;
-import com.google.api.gax.rpc.AbortedException;
-import com.google.api.gax.rpc.BidiStreamingCallable;
-import com.google.api.gax.rpc.ClientStream;
-import com.google.api.gax.rpc.ResponseObserver;
-import com.google.api.gax.rpc.StreamController;
-import com.google.api.gax.rpc.TransportChannelProvider;
+import com.google.api.gax.rpc.*;
```

Comment:
We should just import the packages we need instead of using `*` please.

Reply:
Done.

Reply:
Looks like it is not updated: https://github.com/googleapis/java-bigquerystorage/pull/748/files#diff-683e090fb3d931a66585adce0c23b4266a62b9f47ff80e497a6ebf1e8a1ebe54R32